vizzy/values.yaml

```diff
@@ -0,0 +1,46 @@
+# Default values for vizzy.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 5
+image:
+  repository: scottcbishop/vizzy
+  tag: v3.1.1
+  pullSecret: dockerhub.com
+  pullPolicy: Always
+
+volumeMounts:
+  name: visual-data
+  mountPath: /app/public/visual_images
```

Comment:
If this path is hardcoded, ie. not configurable in the application, it probably shouldn't be configurable here.